File: backend/services/auth_service.py
# backend/services/auth_service.py
class AuthService:
    @staticmethod
    def login(email, password):
        if email == "test@example.com" and password == "password123":
            return True
        return False

    @staticmethod
    def register(email, password):
        # Dummy registration logic
        return True

File: lib/services/auth_service.dart
// backend/services/auth_service.dart

class AuthService {
  static bool login(String email, String password) {
    // Dummy authentication logic
    return email == "test@example.com" && password == "password123";
  }

  static bool register(String email, String password) {
    // Dummy registration logic
    return true;
  }
}
